inspirehep/config.py

```diff
@@ -160,6 +161,11 @@
     }
 })
 
+logger = logging.getLogger()
+handler = logging.StreamHandler()
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
unfortunately this will not work for all loggers but just the defined `logger` in this file.

Reply:
yes, but the code you sent me 
```
handler = logging.StreamHandler()
logging.addHandler(handler)
logging.setLevel(logging.INFO)
```

won't work either

Reply:
I'll add it to the app setup, maybe that'll work @drjova